javaScript/contact.js

```diff
@@ -0,0 +1,14 @@
+const formButton = document.querySelector('#form-button');
+const text = document.getElementById('email').value;
+const regx = /^([a-z\d-]+)@([a-z\d-]+)\.([a-z]{2,8})(\.[a-z]{2,8})?$/;
+
+formButton.addEventListener('click', (a) => {
+  if (!(regx.test(text))) {
+    document.getElementById('error-popup').innerHTML = 'Please enter the valid email i.e., lowercase, abc@gmail.com';
+    document.getElementById('error-popup').style.visibility = 'visible';
+    document.getElementById('error-popup').style.color = 'white';
+    document.getElementById('error-popup').style.textAlign = 'center';
+    document.getElementById('error-popup').style.background = 'red';
+    a.preventDefault();
+  }
+});
```

Comment:
![Screenshot_570](https://user-images.githubusercontent.com/69638013/209116517-91ca78a0-7b96-43ba-ab28-cada6df79f61.png)

- [ ] Kindly make sure that the form can be submitted, currently, I can not submit the form, see the screenshot above ⬆️